[JIT] Remove __torch__ from custom class qualname

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32320 [JIT] Test passing custom class instance to bound method
* #32312 [JIT] Fix returning instance of custom class from method
* **#32301 [JIT] Remove __torch__ from custom class qualname**
* #32324 Temporary workaround for BC test due to schema parser changes

Differential Revision: [D19431645](https://our.internmc.facebook.com/intern/diff/D19431645)

Commit:
[JIT] Remove __torch__ from custom class qualname

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 6c0c7da5:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=6c0c7da55a2b9b3aa43957a54c0d9a09bb5956ed).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4273966) pytorch_linux_backward_compatibility_check_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87162189) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87162189))
<details>
<summary>
<code>Jan 16 18:04:38 caused by: Connection refused (os error 111)</code>
</summary>

```
Jan 16 18:04:38 +++ eval 'extract_trap_cmd ' 
Jan 16 18:04:38 ++++ extract_trap_cmd 
Jan 16 18:04:38 ++++ printf '%s\n' '' 
Jan 16 18:04:38 +++ printf '%s\n' cleanup 
Jan 16 18:04:38 ++ trap -- ' 
Jan 16 18:04:38 cleanup' EXIT 
Jan 16 18:04:38 ++ which sccache 
Jan 16 18:04:38 ++ sccache --stop-server 
Jan 16 18:04:38 Stopping sccache server... 
Jan 16 18:04:38 error: couldn't connect to server 
Jan 16 18:04:38 caused by: Connection refused (os error 111) 
Jan 16 18:04:38 ++ true 
Jan 16 18:04:38 ++ rm /var/lib/jenkins/sccache_error.log 
Jan 16 18:04:38 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Jan 16 18:04:38 ++ SCCACHE_IDLE_TIMEOUT=1200 
Jan 16 18:04:38 ++ RUST_LOG=sccache::server=error 
Jan 16 18:04:38 ++ sccache --start-server 
Jan 16 18:04:38 Starting sccache server... 
Jan 16 18:04:38 ++ sccache --zero-stats 
Jan 16 18:04:38 Compile requests                 0 
Jan 16 18:04:38 Compile requests executed        0 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "[JIT] Remove __torch__ from custom class qualname"


Differential Revision: [D19431645](https://our.internmc.facebook.com/intern/diff/D19431645)

[ghstack-poisoned]

Commit:
Update on "[JIT] Remove __torch__ from custom class qualname"


Differential Revision: [D19431645](https://our.internmc.facebook.com/intern/diff/D19431645)

[ghstack-poisoned]

Commit:
Update on "[JIT] Remove __torch__ from custom class qualname"


Differential Revision: [D19431645](https://our.internmc.facebook.com/intern/diff/D19431645)

[ghstack-poisoned]

Commit:
Update on "[JIT] Remove __torch__ from custom class qualname"


Differential Revision: [D19431645](https://our.internmc.facebook.com/intern/diff/D19431645)

[ghstack-poisoned]

Commit:
Update on "[JIT] Remove __torch__ from custom class qualname"


Differential Revision: [D19431645](https://our.internmc.facebook.com/intern/diff/D19431645)

[ghstack-poisoned]

Commit:
Update on "[JIT] Remove __torch__ from custom class qualname"


Differential Revision: [D19431645](https://our.internmc.facebook.com/intern/diff/D19431645)

[ghstack-poisoned]
